src/main/webapp/assets/js/feedScript.js

```diff
@@ -516,9 +570,11 @@ function applyLogisticFunction(xValue, bounds) {
  * Adds posts to the page (uses mock data).
  * @param {array} posts
  */
-function addPosts(posts) {
+async function addPosts(posts) {
```

Comment:
I'm not 100% familiar with async behavior in JS, does adding `async` here and then calling it with `addPosts()` (no `await`) in `onLoad()` cause it execute as a non-blocking process? Cause if so, cool addition!